src/platform/test/rom-test-main.c

```diff
@@ -0,0 +1,217 @@
+/* Copyright (c) 2013-2022 Jeffrey Pfau
+*  Copyright (c) 2022 Felix Jones
+*
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include <mgba/core/cheats.h>
+#include <mgba/core/config.h>
+#include <mgba/core/core.h>
+#include <mgba/core/serialize.h>
+#include <mgba/internal/gba/gba.h>
+
+#include <mgba/feature/commandline.h>
+#include <mgba-util/vfs.h>
+
+#include <signal.h>
+
+#define ROM_TEST_OPTIONS "S:R:"
+#define ROM_TEST_USAGE \
+	"\nAdditional options:\n" \
+	"  -S SWI           Run until specified SWI call before exiting\n" \
+	"  -R REGISTER      General purpose register to return as exit code\n" \
+
+struct romTestOpts {
```

Comment:
```suggestion
struct RomTestOpts {
```

Sorry, I should have been more explicit; If m is being used as a prefix, the struct name is not initial caps, otherwise it is.